misc.c

```diff
@@ -1699,3 +1700,40 @@ x2dev(s, d)
 	*d = r;
 	return(s);
 }
+
+int
+compare_namespace(ns)
+	char *ns;                /* namespace */
+{
+	char nspath_self[100], nspath_target[100];
```

Comment:
Do not write magic number `100`. Use definitions.

Reply:
Got it.